go/adbc/driver/internal/driverbase/connection.go

```diff
@@ -37,13 +43,20 @@ const (
 	ConnectionMessageOptionUnsupported = "Unsupported connection option"
 	ConnectionMessageCannotCommit      = "Cannot commit when autocommit is enabled"
 	ConnectionMessageCannotRollback    = "Cannot rollback when autocommit is enabled"
+	ConnectionMessageIncorrectFormat   = "Incorrect or unsupported format"
```

Comment:
The error message needs to be explicit that it's an OTel span parent that's incorrect